sphinx-doc/module-hpmc-field.rst

```diff
@@ -0,0 +1,17 @@
+hpmc.field
+------------
+
+.. rubric:: Overview
+
+.. py:currentmodule:: hoomd.hpmc.field
+
+.. autosummary::
+    :nosignatures:
+
+    LatticeField
```

Comment:
I find `hpmc.field.LatticeField` both redundant and not informative. How about `hpmc.field.Harmonic` or `hpmc.field.Einstein`? I'm open to other suggestions.

Reply:
I changed this to `hpmc.field.Harmonic`, and also updated the C++ class to match.